stressberry/cli.py

```diff
@@ -87,13 +110,22 @@ def run(argv=None):
 
     times = []
     temps = []
+    freqs = []
     while t.is_alive():
         times.append(time.time())
-        temps.append(measure_temp(args.temperature_file))
+        temps.append(measure_temp(args.temperature_file, use_vcgencmd))
+        if args.frequency:
```

Comment:
Let's _always_ measure the frequency, and decide at plot time whether to display it.